WalletWasabi/EventSourcing/InMemoryEventRepository.cs

```diff
@@ -0,0 +1,216 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using WalletWasabi.Exceptions;
+using WalletWasabi.Helpers;
+using WalletWasabi.Interfaces.EventSourcing;
+
+namespace WalletWasabi.EventSourcing
+{
+	/// <summary>
+	/// Thread safe without locks in memory event repository implementation
```

Comment:
English grammar doesn't end names with periods.

Reply:
The resistance is because I'm right and you are wrong. You are breaking the English grammar. One doesn't put periods after names. One puts periods after sentences. I'm not gonna follow stupid rules that are wrong.

Reply:
Even if you look at the microsoft document referenced in the guidelines they don't put periods after names in comments. see here: https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#commenting-conventions

`public class ExampleEvents
{
    // A public field, these should be used sparingly
    public bool IsValid;

    // An init-only property
    public IWorkerQueue WorkerQueue { get; init; }

    // An event
    public event Action EventProcessing;

    // Method
    public void StartEventProcessing()
    {
        // Local function
        static int CountQueueItems() => WorkerQueue.Count;
        // ...
    }
}`

Reply:
I'm not gonna be obedient drone. I'll beat you back with truth every single time. If you have a problem with that you can fire me now.